ground/src/main/java/com/google/android/ground/persistence/local/room/dao/UserDao.kt

```diff
@@ -23,7 +23,14 @@ import io.reactivex.Maybe
 
 @Dao
 interface UserDao : BaseDao<UserEntity> {
+
   @Transaction
   @Query("SELECT * FROM user WHERE id = :id")
+  @Deprecated("Use findByIdSuspend")
   fun findById(id: String): Maybe<UserEntity>
```

Comment:
Would it make sense to name this findByIdSync instead and then the non-deprecated function can have the simpler findById name?

Same goes for other functions in this PR

Reply:
I've already marked it as deprecated. So, the plan is to iteratively remove it soon. Calling it sync would be inconsistent as rest of the codebase is still not using that naming convention. Thoughts?

Reply:
Yea I was just thinking about the case where it takes some time for us to get around to removing the deprecated method, then it would be nice if the new function had the cleaner name, but that's a good point that it may be confusing since the rest of the codebase is not using that convention.

SGTM as is

Reply:
Thanks! Can you please stamp approve?

Reply:
Oops, I thought I did